services/language_translation/v1.js

```diff
@@ -13,55 +13,206 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  **/
-
-module.exports = function(RED) {
+module.exports = function (RED) {
   var watson = require('watson-developer-cloud');
   var cfenv = require('cfenv');
+  var fs = require('fs');
+  var temp = require('temp');
 
-  var services = cfenv.getAppEnv().services,
-    service;
+  temp.track();
 
-  var username, password;
+  var username, password, language_translation;
 
-  var service = cfenv.getAppEnv().getServiceCreds(/language translation/i)
+  var service = cfenv.getAppEnv().getServiceCreds(/language translation/i);
 
   if (service) {
     username = service.username;
     password = service.password;
   }
- 
-  RED.httpAdmin.get('/watson-translate/vcap', function(req, res) {
-    res.json(service ? {bound_service: true} : null);
+
+  RED.httpAdmin.get('/watson-translate/vcap', function (req, res) {
+    res.json(service ? {
+      bound_service: true
+    } : null);
+  });
+
+  RED.httpAdmin.get('/watson-translate/models', function (req, res) {
+    language_translation = watson.language_translation({
+      username: username,
+      password: password,
+      version: 'v2'
+    });
+    language_translation.getModels({}, function (err, models) {
+      if (err) {
+        res.json(err);
+      }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Closing curly brace does not appear on the same line as the subsequent block.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2399943306/issues/source?bid=3225115&fileBranchId=3231190#l49)